app/views/user_mentee_applications/_table_row.html.erb

```diff
@@ -9,13 +9,13 @@
 
   <%= render TableCellComponent.new(container_link: mentee_application) do %>
     <%= turbo_frame_tag "#{dom_id(mentee_application)}--note" do %>
-      <%= mentee_application.current_state.note %>
+      <%= mentee_application.current_state&.note %>
     <% end %>
   <% end %>
   
   <%= render TableCellComponent.new(container_link: mentee_application) do %>
     <%= turbo_frame_tag "#{dom_id(mentee_application)}--status" do %>
-      <%= mentee_application.status.humanize %>
+      <%= mentee_application.mentee_application_states.last.status.humanize %>
```

Comment:
isn't this just the `current_state` you want?

Reply:
yup, fixed